src/utils.js

```diff
@@ -0,0 +1,33 @@
+import MAGICNUMBER from './magicnumber.js';
+import { MissionUtils } from '@woowacourse/mission-utils';
+import validation from './validation.js';
+import ERRORMESAGE from './errorMessage.js';
+
+const utils = {
+  pickRandomNumber: () => {
+    return MissionUtils.Random.pickNumberInRange(
+      MAGICNUMBER.STARTINCLUSIVE,
+      MAGICNUMBER.ENDINCLUSIVE,
+    );
+  },
+
+  tryNumvalidation: (tryNum) => {
+    if (!validation.isPositiveInteger(Number(tryNum)))
+      throw new Error(`${ERRORMESAGE.NOT_POSITIVE_INTEGER}`);
+
+    const tryNumArr = String(tryNum).split('');
+    if (validation.hasSpace(tryNumArr))
+      throw new Error(`${ERRORMESAGE.WRONG_NUMBER_FORM_SPACE}`);
+
+    return Number(tryNum);
+  },
+
+  createBoard: (inputArr) => {
+    const board = {};
+    inputArr.map((car) => {
+      board[car] = '';
+    });
+    return board;
+  },
+};
```

Comment:
```suggestion
  createCarStateBoard: (carNames) => {
    const carStateBoard = {};
    carNames.map((car) => {
      carStateBoard[car] = '';
    });
    return carStateBoard;
  },
};
```

네이밍을 조금 수정하면 해당 메서드뿐만 아니라 해당 메서드에 관련된 부분들도 가독성이 올라갈 것 같아요! 🙌

Reply:
board라는 네이밍 자체가 잘못된 것 같네요
최대한 직관적으로 네이밍 한다 했는데 놓친 부분이 많군요